test/bin/local-solver-picks-higher-version.t/repo/packages/base-bigarray/base-bigarray.base/opam

```diff
@@ -0,0 +1,6 @@
+opam-version: "2.0"
```

Comment:
Maybe these could be somehow symlinked? I expect these base packages to be part of a lot of tests so I am worried about them filling up the repo with copies (that might diverge as well).

Reply:
Yeah it would be great, would you be okay if I was to do it in a separate PR?

I'll have to experiment a bit with symlinks and dune rules. The latter would likely be better and more portable of course!

Reply:
Yeah, sure, we can also do that.